Packet++/header/ProtocolType.h

```diff
@@ -352,6 +352,11 @@ namespace pcpp
 	 */
 	const ProtocolType WireGuard = 56;
 
+	/*
+	 * Infiniband protocol
+	 */
+	const ProtocolType Infiniband = 57;
```

Comment:
nit: in all other places InfiniBand has a capital B (InifinBand vs Infiniband).
Same for the doxygen comment